### ISSUE TITLE
Error when using /f who, /f show

### ISSUE THREAD

**Issue by Squirzy** [nsporillo/HCFactions#10]
Hi,

Currently using latest HCFactions and when players use /f show {faction} or /f who {faction} this error message comes up:

https://gyazo.com/81a12f41d3f5531cced1f2ac0154f464

**Comment by nsporillo** [comment]
Looking into this, could you link me your configs?

**Comment by Squirzy** [comment]
Sure: https://pastebin.com/1h9VaNjm

If it helps, when I remove the {online} and {offline} variable from show section of the config the error no longer shows. ({online} and {offline} are removed from the config I linked)

**Comment by nsporillo** [comment]
Okay so you're having a problem with the {online} and {offline} tag variables. You were originally using the default configuration and had this problem?

I'll try and reproduce locally, perhaps there is some sanity checks I could add to the code to gracefully fail under the circumstances you're reporting.

**Comment by Squirzy** [comment]
Yeah, that is correct. Removing the 2 tags has stopped the error from appearing
